app/src/main/java/com/github/libretube/ChannelFragment.kt

```diff
@@ -209,7 +209,7 @@ class ChannelFragment : Fragment() {
                 runOnUiThread {
                     view.findViewById<ScrollView>(R.id.channel_scrollView).visibility = View.VISIBLE
                     view.findViewById<TextView>(R.id.channel_name).text=response.name
-                    view.findViewById<TextView>(R.id.channel_subs).text=response.subscriberCount.formatShort() + " subscribers"
+                    view.findViewById<TextView>(R.id.channel_subs).text=response.subscriberCount.formatShort() + " " + resources.getString(R.string.subscribers)
```

Comment:
Why not have the string as a format literal? In case, a language requires the count to be after the text

Reply:
Good idea, but I don't know how to do this since I'm not so familiar with Kotlin. It would be nice if you could add it :)

Reply:
That's a good idea. But the translators would have to implement into their translated string then? I don't think everybody's familiar with it.

Reply:
> That's a good idea. But the translators would have to implement into their translated string then? I don't think everybody's familiar with it.

@FireMasterK

Reply:
I think it should be fine, if they use weblate since it would show the source string which would contains the formatting character too

For example, I already do this in Piped's web frontend.

Reply:
K, gonna add it then

Reply:
Seems like literals placed in the string.xml don't work because the literal in the strings.xml just gets replaced with nothing in the output while the other parts of the string can be seen (or I'm just too stupid to get it work)